pkg/pipelines/tekton/pac/pac.go

```diff
@@ -36,7 +36,7 @@ func DetectPACInstallation(ctx context.Context, wantedNamespace string) (bool, s
 		return false, "", err
 	}
 
-	_, err = clientPac.Repositories("").List(ctx, metav1.ListOptions{})
+	_, err = clientPac.Repositories(cns).List(ctx, metav1.ListOptions{})
```

Comment:
rationale: unprivileged  user most likely cannot scan all the namespaces, but they most likely can scan current namespace.

Reply:
ack